components/Meta.js

```diff
@@ -34,6 +34,7 @@ export default () => {
         <meta name="theme-color" content="#121212" />
         <title>Carbon</title>
         <link rel="shortcut icon" href="/static/favicon.ico" />
+        <link rel="stylesheet" href="/static/customThemes/one-dark.css" />
```

Comment:
sorry to make this so complicated, we will try to document this process better in the future. Can you actual just do `import '../static/customThemes/one-dark.css'` like the others at the top of this file?